cloud/functions/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "functions",
+  "scripts": {
+    "lint": "eslint --ext .js,.ts .",
+    "build": "tsc",
+    "build:watch": "tsc --watch",
+    "serve": "npm run build && firebase emulators:start  --import data --export-on-exit",
+    "shell": "npm run build && firebase functions:shell",
```

Comment:
@xhanin when I run `npm run shell` I get a `firebase: command not found`
=> I assume there is a missing `firebase` dev dependency (you may have installed `firebase` globally ?)

Reply:
I've never used it actually, it was generated by the initial command. I remove it.

Reply:
fixed through 756ccf7 (I rolled back your changes on b3d56e0 as I made a mistake : it's not `firebase` but `firebase-tools` which was missing0